packages/design-picker/src/components/style.scss

```
@@ -173,3 +173,10 @@
 		}
 	}
 }
+
+/// dark theme styles
+.design-picker.theme-dark {
+	.design-picker__option-name {
+		color: var( --studio-white );
+	}
+}
```

At the moment, the text color used for the "light" theme is the [default text color](https://github.com/Automattic/wp-calypso/pull/52162/files#diff-3c52a0567c030fc5e6767726be99f162eab5d06c2ee122cd1f8713b586c66733R160).

I wonder if it would be better to make that rule explicit for the light theme too. In my opinion, it would make the code more readable, and it would make sure that the text color CSS rules have the same specificity for both dark and light theme

```suggestion

/// dark theme styles
.design-picker.theme-dark {
	.design-picker__option-name {
		color: var( --studio-white );
	}
}

// light theme styles
.design-picker.theme-light {
	.design-picker__option-name {
		color: var( --studio-gray-40 );
	}
}
```